### PR TITLE
LPS-179226 Set ERC with UUID when value is empty

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://liferay.atlassian.net/browse/LPS-179226

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:object
  ci:test:relevant
  ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉
Congrats :smile:

**Comment by marcelabc** [comment]
I'm running the CI again because the last results were from last week.

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Hey @leilanyaragao 
Could you run the integration tests? We must investigate the CI results.
Please, run the integration tests from object-admin-rest-test / object-rest-test / object-test. 
Let me know if you need help :smile:

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -2518,37 +2517,32 @@ private void _testUpdateObjectEntryExternalReferenceCode()
...
-			objectEntry2.getExternalReferenceCode());
+			objectEntry2.getUuid(), objectEntry2.getExternalReferenceCode());
 
-		_objectEntryLocalService.updateObjectEntry(
-			TestPropsValues.getUserId(), objectEntryId2,
-			HashMapBuilder.<String, Serializable>put(
-				"externalReferenceCode", String.valueOf(objectEntryId1)
-			).build(),
-			ServiceContextTestUtil.getServiceContext());
+		ObjectEntry finalObjectEntry2 = objectEntry2;
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2522]: Hey @leilanyaragao
Why did you replace the old code for this new one?
I think we need to maintain the old code that calls the update method. In the parameters, we should update the "externalReferenceCode" with the objectEntry1.getExternalReferenceCode();
What do you think?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**